src/components/CommentList.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByArticle } from "../api";
+import AddComment from "./AddComment";
+
+
+const CommentList = () => {
+
+    const {article_id} = useParams()
+    const [commentList, setCommentList] = useState([])
+    const [isLoading,setIsLoading] = useState(true)
+
+    useEffect(() => {
+        setIsLoading(true)
+        getCommentsByArticle(article_id).then(res => {
+            setCommentList(res.comments)
+            setIsLoading(false)
+        })
+    },[article_id])
+
+
+
+    if (isLoading) {
+        return <p>Loading ...</p>
+    }
+
+    return (
+        <div className="CommentList">
+            <ol>
```

Comment:
Really like the use of OL here seperating comments into individual LI's super clear you're rendering a list